src/main/java/com/google/cloud/tools/appengine/api/versions/VersionsListConfiguration.java

```diff
@@ -16,15 +16,65 @@
 
 package com.google.cloud.tools.appengine.api.versions;
 
-import com.google.cloud.tools.appengine.api.Configuration;
 import javax.annotation.Nullable;
 
-/** Configuration for {@link AppEngineVersions#list(VersionsListConfiguration)}. */
-public interface VersionsListConfiguration extends Configuration {
+/** Plain Java bean implementation of {@link VersionsListConfiguration}. */
+public class VersionsListConfiguration {
+
+  @Nullable private final Boolean hideNoTraffic;
+  @Nullable private final String projectId;
+  @Nullable private final String service;
+
+  private VersionsListConfiguration(
+      @Nullable Boolean hideNoTraffic, @Nullable String projectId, @Nullable String service) {
+    this.hideNoTraffic = hideNoTraffic;
+    this.projectId = projectId;
+    this.service = service;
+  }
+
+  @Nullable
+  public String getService() {
+    return service;
+  }
 
   @Nullable
-  String getService();
+  public Boolean getHideNoTraffic() {
+    return hideNoTraffic;
+  }
 
   @Nullable
-  Boolean getHideNoTraffic();
+  public String getProjectId() {
+    return projectId;
+  }
+
+  public static Builder builder() {
+    return new Builder();
+  }
+
+  public static class Builder {
+    @Nullable private Boolean hideNoTraffic;
+    @Nullable private String projectId;
+    @Nullable private String service;
+
+    private Builder() {}
+
+    public Builder setHideNoTraffic(@Nullable Boolean hideNoTraffic) {
```

Comment:
just wondering (in part so I know for other projects too), but is it the convention for our builders to use `[set]HideNoTraffic` instead of `hideNoTraffic`? Most implementations I've seen use the later.

Reply:
It's what we've been using, also since it's a `Boolean` it can be null (which then delegates to the command line default for that flag).

Reply:
I misread the question. But I'm not opposed to that style, let me see what the styleguide recommendation is.